app/frontend/src/features/profile/hooks/useUpdateHostingPreferences.test.tsx

```diff
@@ -6,11 +6,14 @@ import { act } from "react-test-renderer";
 import { HostingPreferenceData, service } from "service";
 import wrapper from "test/hookWrapper";
 import { addDefaultUser } from "test/utils";
-import { Primitives } from "utils/types";
 
-const getUserMock = service.user.getUser as jest.Mock;
+const getUserMock = service.user.getUser as jest.Mock<
+  ReturnType<typeof service.user.getUser>
+>;
```

Comment:
```suggestion
const getUserMock = service.user.getUser as MockedService<
  typeof service.user.getUser
>;
```
I made the `MockedService` utility type for this purpose in `test/utils` since we need this type assertion for all of the mocked API functions so frequently

Reply:
Ah yes, of course. I shall update it .